plugins/field-dependent-dropdown/test/field_dependent_dropdown.mocha.js

```diff
@@ -0,0 +1,229 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const chai = require('chai');
+const sinon = require('sinon');
+const Blockly = require('blockly');
```

Comment:
this might need to be `blockly/node` since we run mocha tests in node environment, and the default blockly package is browser (not 100% confident on that, maybe our default package is umd, but `blockly/node` is more explicit)